pkg/browsers/console.go

```diff
@@ -181,10 +201,10 @@ const (
 	BrowserDefault
 )
 
-func LaunchConsoleSession(sess Session, labels RoleLabels, service string, region string) error {
+func MakeUrl(sess Session, labels RoleLabels, service string, region string) (error, string) {
```

Comment:
Nitpick, can this be (string, error) as is convention for go functions returning errors, the error is the last value